dev/maven-plugin/src/main/maven/plugin.xml

```diff
@@ -409,6 +409,15 @@
                 <parameter>
                     <name>cassandraEnabled</name>
                 </parameter>
+                <parameter>
+                    <name>kafkaPort</name>
+                </parameter>
+                <parameter>
+                    <name>kafkaAddress</name>
+                </parameter>
```

Comment:
On second thought, let's introduce it with same naming than sbt and we can rename/replace both at once later.